Doc/library/datetime.rst

```diff
@@ -2028,6 +2028,11 @@ For :class:`date` objects, the format codes for hours, minutes, seconds, and
 microseconds should not be used, as :class:`date` objects have no such
 values.  If they're used anyway, ``0`` is substituted for them.
 
+For :meth:`datetime.strptime` class method, default value is ``1900-01-01T00:00:00.000``
+and when using :meth:`strptime`, any components requested in the format string but
+missing from the object itself (in the case of :meth:`datetime.date` and
```

Comment:
This part seems a bit confused. There is no `strptime` method on `date` and `time` objects (at least not yet). The preceding two paragraphs are about `strftime`, which *populates* a string from an object, and those objects may be missing the components specified in the format string.

In `strptime`, you are given a format string and a value string, and the object is populated from the values. The "default value" is talking about what happens when you are given a format string that is missing required components like year, month and day.